src/iOS/Xamarin.iOS/Samples/Map/GetElevationAtAPoint/GetElevationAtAPoint.cs

```diff
@@ -0,0 +1,171 @@
+                InitialViewpoint = new Viewpoint(_observerPoint, 1000000, camera)
+            };
+
+            // Create the marker for showing where the user taps.
+            _elevationMarker = SimpleMarkerSceneSymbol.CreateCylinder(Color.Red, 10, 750);
+
+            // Create the text for displaying the elevation value.
+            _elevationTextSymbol = new TextSymbol("", Color.Red, 20, Esri.ArcGISRuntime.Symbology.HorizontalAlignment.Center, Esri.ArcGISRuntime.Symbology.VerticalAlignment.Middle);
+            _elevationTextGraphic.Symbol = _elevationTextSymbol;
+
+            // Create the base surface.
+            _baseSurface = new Surface();
+            _baseSurface.ElevationSources.Add(new ArcGISTiledElevationSource(_elevationUri));
+
+            // Add the base surface to the scene.
+            myScene.BaseSurface = _baseSurface;
+
+            // Graphics overlay for displaying points.
+            _overlay = new GraphicsOverlay
+            {
+                SceneProperties = new LayerSceneProperties(SurfacePlacement.Absolute)
+            };
+            _mySceneView.GraphicsOverlays.Add(_overlay);
+
+            // Add the scene to the view.
+            _mySceneView.Scene = myScene;
+        }
+
+        public override void LoadView()
+        {
+            // Create the views.
+            View = new UIView();
+
+            _mySceneView = new SceneView();
+            _mySceneView.TranslatesAutoresizingMaskIntoConstraints = false;
+
+            UILabel helpLabel = new UILabel
+            {
+                Text = "Tap to find the elevation for a point.",
+                BackgroundColor = UIColor.FromWhiteAlpha(0f, .6f),
+                TextColor = UIColor.White,
+                TextAlignment = UITextAlignment.Center,
+                TranslatesAutoresizingMaskIntoConstraints = false
+            };
+
+            // Add the views.
+            View.AddSubviews(_mySceneView, helpLabel);
+
+            // Lay out the views.
+            NSLayoutConstraint.ActivateConstraints(new[]{
+                _mySceneView.TopAnchor.ConstraintEqualTo(View.SafeAreaLayoutGuide.TopAnchor),
+                _mySceneView.BottomAnchor.ConstraintEqualTo(View.BottomAnchor),
+                _mySceneView.LeadingAnchor.ConstraintEqualTo(View.LeadingAnchor),
+                _mySceneView.TrailingAnchor.ConstraintEqualTo(View.TrailingAnchor),
+
+                helpLabel.TopAnchor.ConstraintEqualTo(View.SafeAreaLayoutGuide.TopAnchor),
+                helpLabel.LeadingAnchor.ConstraintEqualTo(View.LeadingAnchor),
+                helpLabel.TrailingAnchor.ConstraintEqualTo(View.TrailingAnchor),
+                helpLabel.HeightAnchor.ConstraintEqualTo(40)
+            });
+        }
+
+        public override void ViewDidLoad()
+        {
+            base.ViewDidLoad();
+            Initialize();
+
+            // Handle taps on the scene view for getting elevation.
+            _mySceneView.GeoViewTapped += SceneViewTapped;
```

Comment:
I recommend adding the event subscription in a `ViewWillAppear` overload and unsubscribing in a `ViewDidDisappear` overload. This is a new pattern and doesn't appear to be in the branch your branch is based on. See this example in master: https://github.com/Esri/arcgis-runtime-samples-dotnet/blob/cbd8e6876e033516d818a2d3e8437741bb5e1f47/src/iOS/Xamarin.iOS/Samples/Map/TerrainExaggeration/TerrainExaggeration.cs#L105